modules/ip/datapath/ip_error.c

```diff
@@ -45,11 +46,15 @@ ip_error_process(struct rte_graph *graph, struct rte_node *node, void **objs, ui
 
 		// Get the local router IP address from the input iface
 		iface = ip_output_mbuf_data(mbuf)->input_iface;
-		if (iface == NULL) {
+		gw = ip4_route_lookup(iface->vrf_id, ip->src_addr);
```

Comment:
Just to make sure I understand the fix.
Previously a icmp error would be emitted with the ip address of the interface on which the packet was received.

However, this interface may have no address at all (but I can't picture this but only for loopback ifaces, that grout does not support yet?). Is there other cases?